docs/iris/src/developers_guide/documenting/whats_new_contributions.rst

```diff
@@ -0,0 +1,38 @@
+=================================
+Contributing a "What's New" entry
+=================================
+
+Iris has an aggregator for building an initial "What's New" document for each release, from sections contributed by code authors. This enables the release documentation to be written by the developer most familiar with the change while permitting editing to take place.
+
+Each release should have a directory, "contributions_<versionnumber>". Into this directory can be placed items for inclusion into the release documentation. The filename should be structured as follows:
```

Comment:
If it's mandatory, I'd go with "must". Out of curiosity, if a filename doesn't match that pattern, what happens?